conf/config-default.yaml

```diff
@@ -96,6 +96,9 @@ apisix:
                                   #   see https://github.com/api7/lua-resty-radixtree/#parameters-in-path for
                                   #   more details.
     ssl: 'radixtree_sni'          # radixtree_sni: match route by SNI(base on radixtree)
+  stream:
```

Comment:
I think this section should be added as the same level of the http.

Reply:
By level, it should now be level 2, under `apisix`:
```yaml
apisix:
  stream:
    lua_shared_dict:
      lrucache-lock-stream: 10m
```
`http` is now also at level 2, under `nginx_config`:
```yaml
nginx_config:                     # config for render the template to generate nginx.conf
  http:
    enable_access_log: true        # enable access log or not, default true
```
So I need to adjust the location of the `stream` under `nginx_config`, right?😉